zcash_primitives/CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ and this library adheres to Rust's notion of
     - The `Network` enum, which enables code to be generic over the network type
       at runtime.
   - `zcash_primitives::primitives::Nullifier` struct.
+  - `zcash_primitives::primitives::SaplingIvk` struct. This newtype wrapper is
+    now used where functions previously used undistinguished `jubjub::Fr` values.
```

Comment:
The only place this occured in the `zcash_primitives` crate is in the note encryption APIs, so the second sentence here should be moved to the `Changed` section as an explicit note to that effect.